README.md

```diff
@@ -226,8 +226,10 @@ Options, and their defaults
   // the port the app will start on
   port: 8080,
   // whether or not to run in parallel using all available cpus
-  endpoint: '/batch'
+  endpoint: '/batch',
   // default endpoint path
+  // return the number of cpus hypernova will use
+  getCPUs: undefined
```

Comment:
this should indicate that if null or undefined is passed, that the default will be `cpus => cpus - 1`